DigitalExcellence/dex-frontend
# General low hanging fruit for production

## Description

Fixed the following:

1. 'x' icon showing when input type is set to search (overlapping our search icon)
2. Removed items from account dropdown in navigation bar
3. You need to be authenticated before adding a project now
4. Removed search field in the navigation bar
5. Renamed search on home page to "search project" instead of "Search source"
6. Sort option in project-overview now starts off as Descending instead of Ascending
7. Hitting the enter key when searching/adding a project now triggers submit functionality

## Type of change


## Checklist


## Steps to Test or Reproduce

Outline the steps to test or reproduce the PR here.
These steps will be used during release testing.

'x' icon showing when input type is set to search (overlapping our search icon)
1. Search anywhere and see if it still shows an 'x'
Removed items from account dropdown in navigation bar
1. Sign in and click on your name to see if other options have been removed.

You need to be authenticated before adding a project now
1. Click on add project and enter a url
2. Click on add project and click on add project manually

Removed search field in the navigation bar
1. Check if there is a search field in the nav bar.

Renamed search on home page to "search project" instead of "Search source"
1. Check if search bar on home page says "search project"

Sort option in project-overview now starts off as Descending instead of Ascending
1. Go to project-overview and see if it says Descending instead of Ascending

Hitting the enter key when searching/adding a project now triggers submit functionality
1. Go to the home page and enter something in the search
2. Hit enter to indicate that you want to search

## Link to issue

Closes: #224
